src/tests/ankr.e2e.test.ts

```
@@ -60,7 +64,38 @@ const character: Character = {
     post: [],
   },
   adjectives: [],
-  extends: [],
+};
+
+// Gemini Plugin Implementation
+export const geminiPlugin: Plugin = {
+  name: "gemini",
+  description: "Gemini model provider plugin for ElizaOS",
+  models: {
+    [ModelType.OBJECT_SMALL]: async (runtime: IAgentRuntime, params: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)